### PR TITLE
(refactor): Corrections (grammatical) in service.md file

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav uditgaurav@gmail.com
-  Corrections in  service.md file

### CODE REVIEW

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -432,7 +433,7 @@ You can find more information about `ExternalName` resolution in
 ## Headless Services
 
 Sometimes you don't need load-balancing and a single Service IP.  In
-this case, you can create what are termed “headless” Services, by explicitly
+this case, you can create what is termed “headless” Services, by explicitly
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: This is already correct, no change required. 👎

File: content/en/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -225,7 +225,8 @@ There are a few reasons for using proxying for Services:
 In this mode, kube-proxy watches the Kubernetes master for the addition and
 removal of Service and Endpoint objects. For each Service it opens a
 port (randomly chosen) on the local node.  Any connections to this "proxy port"
-is proxied to one of the Service's backend Pods (as reported via
+are
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: 👍

File: content/en/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -276,9 +277,9 @@ state.
 When accessing a Service, IPVS directs traffic to one of the backend Pods.
 
 The IPVS proxy mode is based on netfilter hook function that is similar to
-iptables mode, but uses hash table as the underlying data structure and works
+iptables mode, but uses a hash table as the underlying data structure and works
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: 👍

File: content/en/docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -276,9 +277,9 @@ state.
...
 in the kernel space.
-That means kube-proxy in IPVS mode redirects traffic with a lower latency than
+That means kube-proxy in IPVS mode redirects traffic with lower latency than
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: No real advantage, but sure. 👍

File: content/en/docs/concepts/services-networking/service.md [hunk 5]
```diff
@@ -310,7 +311,7 @@ about Kubernetes or Services or Pods.
 
 If you want to make sure that connections from a particular client
 are passed to the same Pod each time, you can select the session affinity based
-on client's IP addresses by setting `service.spec.sessionAffinity` to "ClientIP"
+on the client's IP addresses by setting `service.spec.sessionAffinity` to "ClientIP"
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: 👍

File: content/en/docs/concepts/services-networking/service.md [hunk 6]
```diff
@@ -421,7 +422,7 @@ Pods in other Namespaces must qualify the name as `my-service.my-ns`. These name
 will resolve to the cluster IP assigned for the Service.
 
 Kubernetes also supports DNS SRV (Service) records for named ports.  If the
-`"my-service.my-ns"` Service has a port named `"http"` with protocol set to
+`"my-service.my-ns"` Service has a port named `"http"` with the protocol set to
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: 👍

File: content/en/docs/concepts/services-networking/service.md [hunk 7]
```diff
@@ -506,7 +507,7 @@ For example, if you start kube-proxy with the `--nodeport-addresses=127.0.0.0/8`
 If you want a specific port number, you can specify a value in the `nodePort`
 field. The control plane will either allocate you that port or report that
 the API transaction failed.
-This means that you need to take care about possible port collisions yourself.
+This means that you need to take care of possible port collisions yourself.
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: 👍

File: content/en/docs/concepts/services-networking/service.md [hunk 8]
```diff
@@ -550,7 +551,7 @@ Traffic from the external load balancer is directed at the backend Pods. The clo
 
 For LoadBalancer type of Services, when there is more than one port defined, all
 ports must have the same protocol and the protocol must be one of `TCP`, `UDP`
-and `SCTP`.
+,and `SCTP`.
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: Don't begin lines with punctuation.

Add a space after the comma.

File: content/en/docs/concepts/services-networking/service.md [hunk 9]
```diff
@@ -575,7 +576,7 @@ Specify the assigned IP address as loadBalancerIP. Ensure that you have updated
 {{< /note >}}
 
 #### Internal load balancer
-In a mixed environment it is sometimes necessary to route traffic from Services inside the same
+In a mixed environment, it is sometimes necessary to route traffic from Services inside the same
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: No change needed. 👎

File: content/en/docs/concepts/services-networking/service.md [hunk 10]
```diff
@@ -677,7 +678,7 @@ SSL, the ELB expects the Pod to authenticate itself over the encrypted
 connection, using a certificate.
 
 HTTP and HTTPS selects layer 7 proxying: the ELB terminates
-the connection with the user, parse headers and inject the `X-Forwarded-For`
+the connection with the user, parse headers and injects the `X-Forwarded-For`
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]:
```suggestion
the connection with the user, parses headers, and injects the `X-Forwarded-For`
```

File: content/en/docs/concepts/services-networking/service.md [hunk 11]
```diff
@@ -849,7 +850,7 @@ traffic. Nodes without any Pods for a particular LoadBalancer Service will fail
 the NLB Target Group's health check on the auto-assigned
 `.spec.healthCheckNodePort` and not receive any traffic.
 
-In order to achieve even traffic, either use a DaemonSet, or specify a
+In order to achieve even traffic, either uses a DaemonSet, or specify a
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: Proposed change is incorrect but another change is possible.

```suggestion
In order to achieve even traffic, either use a DaemonSet or specify a
```

File: content/en/docs/concepts/services-networking/service.md [hunk 12]
```diff
@@ -1182,7 +1183,7 @@ virtual IP address will simply transport the packets there.
 The Kubernetes project intends to improve support for L7 (HTTP) Services.
 
 The Kubernetes project intends to have more flexible ingress modes for Services
-which encompass the current ClusterIP, NodePort, and LoadBalancer modes and more.
+that encompass the current ClusterIP, NodePort, and LoadBalancer modes and more.
```
Comment (zacharysarah) [inline content/en/docs/concepts/services-networking/service.md]: 👍

**Comment by zacharysarah** [comment]
@uditgaurav Please sign the CLA in order to continue.

File: content/en/docs/concepts/services-networking/service.md (hunk 1 above)
Comment (uditgaurav) [reply content/en/docs/concepts/services-networking/service.md]: okay

File: content/en/docs/concepts/services-networking/service.md (hunk 8 above)
Comment (uditgaurav) [reply content/en/docs/concepts/services-networking/service.md]: okay sure

**Comment by uditgaurav** [comment]
@zacharysarah Thanks for the review I have addressed your suggestions.

**Comment by zacharysarah** [comment]
@uditgaurav Welcome to Kubernetes!

/lgtm
/approve

**Pull request merged**